### ISSUE TITLE
[BUG] Inserting an Ref does not work

### ISSUE THREAD

**Issue by boonkerz** [Kephson/paste_reference#20]
## Summary / Description

Insert an Ref does not work

## Version

2.0.4
11.5.24

### Additional

Console:

```
Uncaught TypeError: $closestElementWithLanguage.data is not a function
    at DragDrop.default.onDrop (PasteReferenceDragDrop.js?bust=1678439888:190:59)
    at PasteReferenceOnReady.js?bust=1678439888:169:26
DragDrop.default.onDrop @ PasteReferenceDragDrop.js?bust=1678439888:190
(anonym) @ PasteReferenceOnReady.js?bust=1678439888:169
postMessage (async)
send @ MessageUtility.js?bust=1678439888:13
addElement @ ElementBrowser.js?bust=1678439888:13
insertElement @ ElementBrowser.js?bust=1678439888:13
(anonym) @ BrowseDatabase.js?bust=1678439888:13
(anonym) @ RegularEvent.js?bust=1678439888:13
```

**Comment by Mordef** [comment]
Composer-Patch: 
[ehaerer-paste-reference-resources-public-javascript-pastereferencedragdrop-js.patch](https://github.com/Kephson/paste_reference/files/11251122/ehaerer-paste-reference-resources-public-javascript-pastereferencedragdrop-js.patch)

**Comment by Kephson** [comment]
closed with new release 2.0.5
